src/scripts/main.js

```diff
@@ -1 +1,14 @@
 'use strict';
+
+const mainPhoto = document.querySelector('#largeImg');
+const images = document.querySelectorAll('.list-item');
```

Comment:
Ensure that elements with the class 'list-item' are intended to be images and that this class name is not too generic which might affect other list items that are not images.